### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by yuxxeun** [dvtng/react-loading-skeleton#134] [General]
So i just use react-loading-skeleton to my nextjs app, but nothing change happen, did i write it well? Or i miss something? I write it in layout anyway
![IMG_20220206_113139](https://user-images.githubusercontent.com/50067865/152668644-a66069ed-b5eb-4d77-9ea3-903f3df99028.jpg)

**Comment by srmagura** [comment]
Hey @yuxxeun, `<Nav />` and `<Hero />` are always going to be "truthy", so your skeletons will never render. The `show={loading}` part also looks incorrect.

Usually, you will show the skeletons if you are waiting for an API call to complete. I don't see any API calls in your code snippet.

**Reply by yuxxeun** [reply]
so skeleton loading just work when my app call API, right? and i could remove show attribute.
indeed, my app hasn't called any API yet.
ok, i'll try it again with an API.
